spa/src/components/common/Modal/AddPageModal/AddPageModal.test.tsx

```diff
@@ -0,0 +1,103 @@
+import { axe } from 'jest-axe';
+import { fireEvent, render, screen, waitFor } from 'test-utils';
+
+import userEvent from '@testing-library/user-event';
+import AddPageModal from './AddPageModal';
+
+const revenuePrograms = [
+  { id: '1', name: 'rp-mock-1' },
+  { id: '2', name: 'rp-mock-2' },
+  { id: '3', name: 'rp-mock-3' }
+];
+
+const onClose = jest.fn();
+const onClick = jest.fn();
+
+describe('AddPageModal', () => {
+  const renderComponent = (outerError?: string, loading?: boolean) =>
+    render(
+      <AddPageModal
+        open={true}
+        onClose={onClose}
+        onClick={onClick}
+        revenuePrograms={revenuePrograms}
+        outerError={outerError}
+        loading={loading}
+      />
+    );
```

Comment:
For consistency across tests...

```suggestion
function tree(props?: Partial<AddPageModalProps>)
    render(
      <AddPageModal
        open
        onClose={onClose}
        onClick={onClick}
        revenuePrograms={revenuePrograms}
        {...props}
      />
    );
```